src/models/table_type.rs

```diff
@@ -0,0 +1,36 @@
+use serde::{Deserialize, Serialize};
+use std::fmt::Display;
+use std::str::FromStr;
+
+#[derive(
+    Default, Clone, Copy, Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Serialize, Deserialize,
+)]
+pub enum TableType {
+    #[default]
+    A,
+    B,
+    C,
+}
+
+impl Display for TableType {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            TableType::A => write!(f, "A"),
+            TableType::B => write!(f, "B"),
+            TableType::C => write!(f, "C"),
+        }
+    }
+}
+
+impl FromStr for TableType {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
```

Comment:
Is the case important here? Maybe we could be matching on upper and lower case letters?

Reply:
this is indeed leftover from copied part of the code from openapi generator

Reply:
deleted from_str